tox.ini

```diff
@@ -107,6 +107,7 @@ commands =
 
 [testenv:coverage]
 description = Run unit tests with coverage
+allowlist_externals = mkdir
```

Comment:
I have a separate email thread open with our TIOBE contact about the coverage file location, so might be able to simplify things there. If I understand correctly, they don't like it in the top level because then they search the entire tree for it (I have no idea why there's no early exit or ability to just specify exactly where it is). Using `.coverage` is particularly annoying since that's the default filename for the data file, so I hope to at least avoid that. Anyway, something for a future PR.